ARCHITECTURE.md

```diff
@@ -1,25 +1,25 @@
 # Architecture
 
-General architecture description of gotd project.
+General architecture and description of gotd.
```

Comment:
Initially it should be something like "description of architecture", but this wording is OK too